docs/design/resource-strategy-fit-scheduling.md

```diff
@@ -72,6 +75,61 @@ node score:
 finalScoreNode = [(weight1 * resource1) + (weight2 * resource2) + … + (weightN* resourceN)] /(weight1+weight2+ … +weightN)
 ```
 
+## Syntax Rules
+### Wildcard Syntax Support
+- Only suffix wildcard patterns are supported (e.g., `nvidia.com/gpu/*`)
+- Invalid patterns are filtered out during configuration parsing:
+    - Single asterisk (`*`)
+    - Asterisk in the middle (`vendor.*/gpu`)
+    - Asterisk at the beginning (`*/gpu`)
+    - Multiple asterisks (`vendor.com/**`)
+
+### Matching Priority
+1. **Exact match** takes highest priority
+2. **Longest prefix match** for wildcard patterns
+
+**Configuration Example:**
+```yaml
+resources:
+  nvidia.com/gpu-v100:     # Exact match - highest priority
+    type: MostAllocated
+    weight: 3
+  nvidia.com/gpu/*:        # Wildcard match - covers other NVIDIA GPUs
```

Comment:
How many match pattern will be supported, only wildcard? Do we need to  add other patterns or we only support one pattern?